### PR TITLE
Feature/6939 vaccination certificate toolkit api

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
Adds API and Stub for HealthCertificateToolkit

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6939

### CODE REVIEW

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    func decompress(_ data: Data) -> Data {
+        return Data()
+    }
+
+    func decodeCOSEPayload(_ data: Data) -> Data {
+        return Data()
+    }
+
+    func decodeJSON(_ data: Data) -> Data {
```
Comment (mlenkeit) [inline src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift:55]: semantically, this is either `decodeCBOR` or `extractJSON`. 

`decodeJSON` implies that _JSON_ is the input for the funktion

File: src/xcode/ENA/HealthCertificateToolkit/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ElectronicHealthCertificate
```
Comment (nickguendling) [inline src/xcode/ENA/HealthCertificateToolkit/README.md:1]: I guess this was the old package name

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift [hunk 3]
```diff
@@ -0,0 +1,89 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+
+import Foundation
+import Compression
```
Comment (nickguendling) [inline src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift:7]: Still needed here?

**Review by KaiTeuber — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

File: src/xcode/ENA/HealthCertificateToolkit/README.md (hunk 2 above)
Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/README.md:1]: right, renamed

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift:55]: Renamed. I think I will need to refactor a lot of naming, as soon as I get more familiar with the CBOR things.

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift (hunk 3 above)
Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateToolkit.swift:7]: No. removed.

**Pull request merged**